server/src/test/java/org/bonitasoft/web/rest/server/api/bpm/process/ProcessInstantiationResourceTest.java

```diff
@@ -42,6 +35,15 @@
 import org.restlet.data.Status;
 import org.restlet.resource.ServerResource;
 
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 @RunWith(MockitoJUnitRunner.class)
 public class ProcessInstantiationResourceTest extends RestletTest {
```

Comment:
shouldn't there be a test that checks that deleteFiles has been called effectively ?